src/group/components/GroupsHomeCard.js

```diff
@@ -102,24 +101,25 @@ const GroupsHomeCard = ({ groups }) => {
 
   return (
     <HomeCard
-      aria-labelledby="groups-list-title"
-      sx={{ flexDirection: 'column' }}
+      title={t('group.home_default_title')}
+      action={{
+        label: t('group.home_connect_label'),
+        to: '/groups',
+      }}
+      contentBackgroundColor="white"
+      ariaLabel="groups-list-title"
```

Comment:
Issue: To localize this you can do:
```suggestion
      ariaLabel={t('group.aria_label')}
```

Reply:
Ohh, I think this is an id, maybe this property should be called `ariaLabelledBy`